<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>330</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Gold Rate:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="GoldRate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Silver Rate:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SilverRate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Opening Cash:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="OpnCash">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
